logging/cluster-logging-external.adoc

```diff
@@ -14,16 +14,6 @@ To send logs to other log aggregators, you use the {product-title} Cluster Log F
 ====
 To send audit logs to the default internal Elasticsearch log store, use the Cluster Log Forwarder as described in xref:../logging/config/cluster-logging-log-store.adoc#cluster-logging-elasticsearch-audit_cluster-logging-store[Forward audit logs to the log store].
 ====
-
-When you forward logs externally, the Red Hat OpenShift Logging Operator creates or modifies a Fluentd config map to send logs using your desired protocols. You are responsible for configuring the protocol on the external log aggregator.
```

Comment:
Could you double check if this line needs to be removed?

Reply:
@alanconway / @periklis / @jcantrill ^ - Thoughts?